From 0ac3b4cd563b9742d6ede221313c752f4c748fc4 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 8 Jun 2023 09:15:47 +0200
Subject: [PATCH] arm64: defconfig: enable CRC32 support

Enable STM32 CRC32 support on 64 bits platform.

Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: Id0b6af271aab14ba4ab688378ca39402b40ebffd
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/311510
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/360577
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index bc55d43ac332..f6a2c8a68781 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -3,6 +3,7 @@ CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_BACKLIGHT_GPIO=y
 CONFIG_CLKSRC_STM32_LP=y
 CONFIG_COUNTER=m
+CONFIG_CRYPTO_DEV_STM32_CRC=m
 CONFIG_DRM=y
 CONFIG_DRM_BRIDGE=y
 CONFIG_DRM_DISPLAY_CONNECTOR=y
-- 
2.39.5

